lib/nostrum/shard/event.ex

```diff
@@ -20,42 +20,41 @@ defmodule Nostrum.Shard.Event do
 
     if payload.t == :READY do
       Logger.info("READY")
-      %{state | session: payload.d.session_id}
+      {%{state | session: payload.d.session_id}, []}
```

Comment:
Since ready events now include the URL Discord wants us to use for [future resumes](https://discord.com/developers/docs/topics/gateway-events#ready-ready-event-fields), should we add that to the shard data here?